src/lib/Util.js

```diff
@@ -32,4 +32,22 @@ export default class Util {
 
     element.style.display = "block";
   }
+
+  static closeNavbarEventListeners(hamburgerBtn, mainElement) {
+    let navOpen = false;
+
+    hamburgerBtn.addEventListener("click", () => {
+      if (navOpen === false) {
+        navOpen = true;
+        mainElement.addEventListener("click", () => {
+          hamburgerBtn.click();
+        });
+      } else {
+        navOpen = false;
+        mainElement.removeEventListener("click", () => {
+          click(hamburgerBtn);
+        });
+      }
+    });
+  }
```

Comment:
This logic need be discussed. 
1. We should not `addEventListener` and `removeEventListener` repetitively.
2. Instead, we should use event delegation
3. We can check the attribute to get whether the navbar is collapsed or not